sci-mathematics/glucas/glucas-2.9.0.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="A client to make primality test for Mersenne numbers"
+DESCRIPTION="Glucas is related to GIMPS effort to search the biggest primes \ 
+ever found."
+HOMEPAGE="https://sourceforge.net/projects/glucas/"
+COPYRIGHT="2001 gbvalor"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/glucas/Glucas-$portVersion.tar.gz"
+CHECKSUM_SHA256="fde422220245788c0d8aeaec6a89f26e849905b1417c5e6ff1a01f56c595cadc"
+PATCHES="glucas-2.9.0.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	glucas = $portVersion
+	cmd:Glucas = $portVersion
+	"
+REQUIRES="
+	haiku
+	"		
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:automake
```

Comment:
```diff
-	cmd:aclocal
-	cmd:automake
```

Please only define what's being used.

Reply:
I get an error when I remove cmd:aclocal. So I only removed cmd:automake.

Reply:
1) `cmd:aclocal` and `cmd:automake` are provided by the same package, `automake`.
2) The configure script calls both aclocal and automake.
3) We should not, imo, assume that 1) will always be true. `cmd:aclocal` could very well be moved to a stand-alone subpackage.

We should therefore keep both `cmd:aclocal` and `cmd:automake`.

That said, since having only one of these works fine, I don't object dropping `cmd:automake`. I just think it would be nicer to keep it.

@nishanth1232: if you can check that having `cmd:automake` and dropping `cmd:aclocal` would also work.
What happens is that when haikuporter sees the `cmd:aclocal` it activates the automake package because that's the pacakge which provides `cmd:aclocal`. Similarly, when haikuporter sees the `cmd:automake` it (also) activates the automake package because that's the pacakge which provides `cmd:automake`. That's why I think the recipe looks nicer with both `cmd:aclocal` and `cmd:automake`.

@alaviss: Would you agree with keeping both `cmd:aclocal` and `cmd:automake` ?

@nishanth1232: We'll also need `cmd:makeinfo`.